pages/services/kubernetes/2.1.0-1.12.3/release-notes/index.md

```diff
@@ -46,11 +45,11 @@ excerpt: Release notes for DC/OS Kubernetes version 2.1.0-1.12.3
 
 ## Documentation
 
-* Add an [Overview](/services/kubernetes/2.1.0-1.12.3/overview/) page explaining in detail what changed since the 1.x series of releases.
-* Add a [CLI](/services/kubernetes/2.1.0-1.12.3/cli/) page detailing the new Mesosphere Kubernetes Engine CLI.
-* Merged `Advanced Installation` page merging its content into [Customizing your Installation](/services/kubernetes/2.1.0-1.12.3/operations/customizing-install/).
-* Add a [Private Docker Registry](/services/kubernetes/2.1.0-1.12.3/operations/private-docker-registry/) page explaining how to configure it.
+* Add an [Overview](../overview) page explaining in detail what changed since the 1.x series of releases.
+* Add a [CLI](../CLI) page detailing the new Mesosphere Kubernetes Engine CLI.
+* Merged `Advanced Installation` page merging its content into [Install and Customize](../install-and-customize).
```

Comment:
And this one?

Reply:
Did we change the layout of the docs again or is this accidental because the 2 repos were out of sync?

Reply:
out of sync